framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilderImpl.java

```diff
@@ -63,4 +83,58 @@ public WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withT
         envVariables.put(OpenShiftTemplateConstants.TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL, Long.toString(timerServiceDataStoreRefreshInterval.toMillis()));
         return this;
     }
+
+    @Override
+    public WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder deploySSO(boolean deploySSO) {
+        this.deploySSO = deploySSO;
+        if (deploySSO) {
+            envVariables.put(OpenShiftTemplateConstants.SSO_USERNAME, DeploymentConstants.getSSOServiceUser());
+            envVariables.put(OpenShiftTemplateConstants.SSO_PASSWORD, DeploymentConstants.getSSOServicePassword());
+        }
+        return this;
+    }
+
+    @Override
+    public WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withBusinessCentralMavenUser(String user, String password) {
+        envVariables.put(OpenShiftTemplateConstants.BUSINESS_CENTRAL_MAVEN_USERNAME, user);
+        envVariables.put(OpenShiftTemplateConstants.BUSINESS_CENTRAL_MAVEN_PASSWORD, password);
+        return this;
+    }
+
+    @Override
+    public WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withHttpWorkbenchHostname(String http) {
```

Comment:
The parameter name is a bit confusing. Is it really hostname (like google.com - in which case it shouldn't matter what protocol we're using) or the whole url (in which case the distinction http/https matters)? Maybe just rename the parameter names in this and similar classes to `(String hostname)`